applications/FSIApplication/python_scripts/partitioned_embedded_fsi_base_solver.py

```diff
@@ -72,6 +52,45 @@ def __init__(self, model, project_parameters):
         KratosMultiphysics.Logger.PrintInfo('PartitionedEmbeddedFSIBaseSolver', 'Fluid solver construction finished')
         KratosMultiphysics.Logger.PrintInfo('PartitionedEmbeddedFSIBaseSolver', 'Partitioned embedded FSI base solver construction finished')
 
+    @classmethod
+    def GetDefaultParameters(cls):
+
+        # Note that only the coupling settings are validated
+        # The subdomain solver settings will be validated while instantiating these
+        default_settings = KratosMultiphysics.Parameters("""
+        {
+            "echo_level": 0,
+            "parallel_type": "OpenMP",
+            "solver_type": "partitioned_embedded",
+            "coupling_scheme": "dirichlet_neumann",
+            "structure_solver_settings": {
+            },
+            "fluid_solver_settings":{
+            },
+            "coupling_settings":{
+                "coupling_strategy_settings": {
+                    "abs_cut_off_tol": 1e-06,
+                    "solver_type": "MVQN",
+                    "w_0": 0.5
+                },
+                "nl_max_it": 30,
+                "nl_tol": 1e-07,
+                "structure_interfaces_list": []
+            }
+        }""")
+
+        default_settings.ValidateAndAssignDefaults(default_settings)
```

Comment:
this is missing the parameters from the baseclass

Reply:
Yes! Thanks for spotting.

Reply:
Now should be fine.

Reply:
thx

You are still blocking though :D (we anyway have to wait for @loumalouomega)